Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build Node app
-FROM node:18-bullseye
+FROM node:22-bullseye
```

Comment:
Why bookworm in transition and bullseye in evolution?

Reply:
Because of trRouting/OSRM dependency in Transition, it's harder to upgrade.

Reply:
Oh, I guess we just did not update it

Reply:
I'll add a commit to update to bookworm